tests/cross/src/test/scala/tests/pc/ExtractMethodSuite.scala

```diff
@@ -0,0 +1,89 @@
+package tests.pc
+
+import java.net.URI
+
+import scala.meta.internal.jdk.CollectionConverters._
+import scala.meta.internal.metals.CompilerRangeParams
+import scala.meta.internal.metals.TextEdits
+
+import munit.Location
+import munit.TestOptions
+import org.eclipse.{lsp4j => l}
+import tests.BaseCodeActionSuite
+
+class ExtractMethodSuite extends BaseCodeActionSuite {
+
+  checkEdit(
```

Comment:
Let's add more tests here so that we can test both Scala 2 and Scala 3 with compat options.